man/man8/facter.8

```diff
@@ -45,14 +78,48 @@ hardwaremodel => x86_64
 .IP "" 0
 .
 .P
-Display a single fact:
+Display a single structured fact:
 .
 .IP "" 4
 .
 .nf
 
-$ facter kernel
-Linux
+$ facter processors
+{
+  count => 2,
+  isa => "x86_64",
+  models => [
+    "Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz",
+    "Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz"
+  ],
+  physicalcount => 2
+}
+.
+.fi
+.
+.IP "" 0
+.
+.P
+Display a single fact nested within a structured fact:
+.
+.IP "" 4
+.
+.nf
+$ facter processors.isa
+x86_64
+.
+.fi
+.
+.IP "" 0
+.
+.P
+Display a single legacy fact:
```

Comment:
Yeah, we want to show that those facts are still available, but not displayed in the output because they'd show duplicated data from the newer structured facts.